src/app/page.tsx

```diff
@@ -16,8 +16,10 @@ export default function Home() {
   const [disabled, setDisabled] = useState(true)
   const [ageGroup, setAgeGroup] = useState('')
   const [gender, setGender] = useState('')
+  const [quote, setQuote] = useState('')
 
   useEffect(() => {
+    setQuote(quotes[Math.floor(Math.random() * 7)])
```

Comment:
This fires every time the user changes a selection. Can we move this to a separate useEffect method and have an empty state array => 
`useEffect(() => setQuote(quotes[Math.floor(Math.random() * 7)]), [])`